packages/manager/modules/pci/src/components/project/instance/instance.class.js

```diff
@@ -157,4 +157,8 @@ export default class Instance {
     }
     return 'X.X.X.X';
   }
+
+  canBeHardRebooted() {
+    return ['ACTIVE', 'SHUTOFF'].includes(this.status);
```

Comment:
Extract to constants ? :)

Reply:
Instances status have been written as strings everywhere in the entire pci module. This would require a lots of refactoring, i guess it's out of the scope of this PR but i'll keep it in mind